test/integration_tests/wav2vec2_pipeline_test.py

```diff
@@ -61,7 +61,6 @@ def test_pretraining_models(bundle):
 )
 def test_finetune_asr_model(
         bundle,
-        lang,
         expected,
         sample_speech,
         ctc_decoder,
```

Comment:
where do these come from?

Reply:
It's defined in `conftest.py`, and PyTest will substitute it automatically. 

https://github.com/pytorch/audio/blob/184466a904c45d3a7e84629bff647b874f5448d5/test/integration_tests/conftest.py#L30-L32